src/ucs/config/global_opts.c

```diff
@@ -78,6 +79,10 @@ static ucs_config_field_t ucs_global_opts_table[] = {
   ucs_offsetof(ucs_global_opts_t, log_file),
   UCS_CONFIG_TYPE_STRING},
 
+ {"LOG_FILE_SIZE", "inf",
+  "The maximal size of log file. The file is re-written after the maximal size is reached.",
```

Comment:
maybe mention that it is being rewtitten in the circular manner

Reply:
done